server/main.go

```diff
@@ -344,6 +344,18 @@ func main() {
 				if err := rtcPeer.HandleMessage(messageIn.Message); err != nil {
 					fmt.Println("malformed rtc message from", seq, string(messageIn.Message), err)
 				}
+			case "chat":
+				var chatMessage struct {
+					Message json.RawMessage `json:"message"`
+				}
+				err := json.Unmarshal(msg.Body, &chatMessage)
+				if err != nil {
+					fmt.Println(err)
+					break
+				}
+				if err := rtcPeer.HandleMessage(chatMessage.Message); err != nil {
+					fmt.Println("malformed chat message from", seq, string(chatMessage.Message), err)
+				}
```

Comment:
I have no idea what I'm doing, just kind of copy/paste guessing from other cases here. It looks like I'm posting messages to the server, but I get this in the terminal:

```
malformed chat message from 2 "tttt" json: cannot unmarshal string into Go value of type []json.RawMessage
```

… just not sure what to do about that error.